DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Administrator/AdministratorController.cs

```diff
@@ -6,6 +6,7 @@
     using DigitalLearningSolutions.Data.DataServices;
     using DigitalLearningSolutions.Data.DataServices.UserDataService;
     using DigitalLearningSolutions.Data.Models.Common;
+    using DigitalLearningSolutions.Data.Models.User;

```

Comment:
This reference was added but my Visual Studio seems to think it is unused. Running Cleanup Code should get rid of any unused references here.

Reply:
I ran the clearn up code from ReSharper